suricata/update/rule.py

```diff
@@ -154,6 +154,10 @@ def find_opt_end(options):
 
     while True:
         i = options[offset:].find(";")
+        if(options[offset:i+1].find("classtype")!=-1):
+            j=options[offset+2:].find(" ")
+            if(j!=-1):
+                i=j
```

Comment:
When I gave an example about the missing semicolon after `classtype` var, I did not mean that it will be the only case. A user can miss a semicolon anywhere in the rule.
To be clear, we need to catch the Exception that is caused in such a case and exit with a clean error instead of an ugly traceback.
Is this helpful?